contracts/peripheral/YieldStreamer.sol

```diff
@@ -315,16 +288,15 @@ contract YieldStreamer is IYieldStreamer, YieldSplitter, OlympusAccessControlled
             }
         }
 
-        uint256 feeToDao = (totalGOHM * feeToDaoPercent) / 1000000;
+        uint256 feeToDao = (totalGOHM * feeToDaoPercent) / sixDecimalMaxNumber;
         IERC20(gOHM).safeTransfer(authority.governor(), feeToDao);
 
-        uint256 totalOhmToSwap = staking.unwrap(address(this), totalGOHM - feeToDao);
-        staking.unstake(address(this), totalOhmToSwap, false, false);
+        uint256 totalOhmToSwap = staking.unstake(address(this), totalGOHM - feeToDao, false, false);
```

Comment:
I saw you got rid of the other unstake interaction, but should this one have _rebasing set to true like the audit mentioned?

Reply:
setting _rebasing to true means unstake sOhm to ohm. By putting false i can directly unstake gOhm to ohm. which is more efficient.

Reply:
My implementation before the audit unwrapped gOhm to sOhm then unstaked sOhm. Now knowing this its just better i directly unstake gOhm to ohm instead of sOhm first.

Reply:
ok great